core/net/ipv6/uip-nd6.c

```diff
@@ -662,7 +662,8 @@ rs_input(void)
         }
         if(memcmp(&nd6_opt_llao[UIP_ND6_OPT_DATA_OFFSET],
             lladdr, UIP_LLADDR_LEN) != 0) {
-          uip_ds6_nbr_t nbr_data = *nbr;
+          uip_ds6_nbr_t nbr_data;
+          nbr_data = *nbr;
```

Comment:
Is this change really necessary. cc65 is supposed to support variable definitions with initialization.